src/cron/updateCurrentStreak.ts

```diff
@@ -1,17 +1,13 @@
 import { Cron } from './cron';
 import { User, UserStreak } from '../entity';
 import { checkUserStreak, clearUserStreak } from '../common';
-import { opentelemetry } from '../telemetry/opentelemetry';
+import { counters } from '../telemetry';
 
 const cron: Cron = {
   name: 'update-current-streak',
   handler: async (con, logger) => {
     try {
-      const streakCounter = opentelemetry.metrics
-        .getMeter('api-bg')
-        .createCounter('streak_update', {
-          description: 'How many streaks get updated',
-        });
+      const streakCounter = counters.cron.streakUpdate;
```

Comment:
That's correct

Reply:
It might work for a while as long as we still also push metrics via GCP. But if all works with prometheus, we'll remove that metric export method.

And it hasn't worked till now anyways, as cron never had trace/metrics enabled.